tests/cases/compiler/typeAliasDeclareKeywordNewlines.ts

```diff
@@ -0,0 +1,19 @@
+var declare: any, type: any;

```

Comment:
Consider giving these distinct types

```suggestion
var declare: string, type: number;
```

Reply:
Why?

Reply:
Probably so that the types baselines can show that we associate the types of these correctly, I think?

Reply:
That makes sense. I pushed an update with this change.